db/migrate/20230126144504_add_author_reference_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorReferenceToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] You have done a great job adding the user's reference to the likes table. However, this will result in having a user tag/name and according to the project requirements ERD, the post is supposed to reference the user's table as an author. You could instead do it this way: ```add_reference :likes, :author, foreign_key: {to_table: :users}```

![image](https://user-images.githubusercontent.com/86318284/214903555-caf4a1a6-b5eb-4f36-8a7e-27ad2aa0c8a7.png)